### PR TITLE
chore: update grafana dashboards and add backup instructions

### LINKED ISSUE #3: Remove severity filters from all dashboards
## Description

Ensure that the `severity` property is no longer used in error type events in the Grafana dashboards. This change follows the decision made in SRGSSR/pillarbox-documentation#107. The Grafana dashboards must be updated to remove any filters by "severity", and the "severity" property should no longer be displayed in error events.

## Acceptance Criteria

- Remove any filters by `severity` from all dashboards and panels.
- Ensure the `severity` property is no longer displayed in any error-related panels or visualizations.

### PR DESCRIPTION
**Pull request by jboix**
## Description

Resolves #3 by removing severity filters from Grafana dashboards.

## Changes Made

- Added backing up instructions under `docs/DASHBOARD_EXPORT_GUIDE.md`
- Updated submodules to latest version.
- Updated dashboards in provisioning to align with production state.
- Adapted Grafana version in Dockerfile for consistent local testing.
- Use aliased index in datasource provisioning (see SRGSSR/pillarbox-monitoring-transfer#16)

## Checklist

- [x] I have followed the project's style and contribution guidelines.
- [x] I have performed a self-review of my own changes.
- [x] I have made corresponding changes to the documentation.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by defagos — APPROVED** [review]

**Comment by defagos** [comment]
Maybe documenting the manual way to backup a dashboard could be valuable as well? (Share > Export > Save to file)

File: docs/DASHBOARD_EXPORT_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+### 1. Open the Browser Console
+
+- Open your Grafana instance and log in if prompted by SSO.
+- Press **F12** (or right-click and select **Inspect**) to open Developer Tools.
+- Navigate to the **Console** tab.
+
+### 2. Load JSZip Library
+
+- To manage multiple files in a zip, we’ll use the [JSZip library](https://stuk.github.io/jszip/).
+  Copy and paste the following code into the console and press **Enter**:
```
Comment (MGaetan89) [inline docs/DASHBOARD_EXPORT_GUIDE.md:17]:
```suggestion
  Copy and paste the following code into the console and press <kbd>Enter</kbd>:
```

File: docs/DASHBOARD_EXPORT_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  ```javascript
+  var script = document.createElement('script');
+  script.src = 'https://cdn.jsdelivr.net/npm/jszip@3.10.1/dist/jszip.min.js';
+  document.head.appendChild(script);
+  ```
+
+- Verify that JSZip is loaded by typing `JSZip` into the console and pressing **Enter**. You should
```
Comment (MGaetan89) [inline docs/DASHBOARD_EXPORT_GUIDE.md:25]:
```suggestion
- Verify that JSZip is loaded by typing `JSZip` into the console and pressing <kbd>Enter</kbd>. You should
```

File: docs/DASHBOARD_EXPORT_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  see an object or function returned.
+
+### 3. Run the Export Script
+
+- Copy and paste the following code into the console and press **Enter**:
```
Comment (MGaetan89) [inline docs/DASHBOARD_EXPORT_GUIDE.md:30]:
```suggestion
- Copy and paste the following code into the console and press <kbd>Enter</kbd>:
```

**Review by MGaetan89 — APPROVED** [review]

File: pillarbox-monitoring-grafana/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Dockerfile for grafana
-FROM grafana/grafana:11.0.0
+FROM grafana/grafana:10.4.1
```
Comment (MGaetan89) [inline pillarbox-monitoring-grafana/Dockerfile:2]: 10.4.1 is the Grafana version used on AWS, correct?

Comment (jboix) [reply pillarbox-monitoring-grafana/Dockerfile:2]: Currently yes.

**Pull request merged**